sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyAsyncClient.java

```diff
@@ -161,7 +158,23 @@ public class CryptographyAsyncClient {
         this.keyId = keyId;
         this.pipeline = pipeline;
         this.implClient = new CryptographyClientImpl(keyId, pipeline, version);
-        this.key = null;
+
+        LocalKeyCryptographyClient localClient = null;
+
+        try {
+            localClient = retrieveJwkAndInitializeLocalClient(this.implClient);
```

Comment:
Retrieving the jwk requires a service call. This is made from the constructor of the class which we generally don't do as it can't be async. Should we lazily call the service to get the jwk when the first operation on this client is called?

Reply:
Yeah, we can do that. We had been doing it before but I thought we could do it in the constructor.

Reply:
Moved this to method calls instead of the constructor.